candrepa1/take-home-assignment
# Take home assignment

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



Part of the tags inside of App.js were moved into smaller components. Two radio buttons were added for the user to pick between uppercase mode and lowercase mode. The logic to turn the text into uppercase or lowercase depending on the mode was added to App.js. 

Some tests were made to check the rendering of tags inside each component. 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
| ✓   | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria



* From your terminal, clone this repo running git clone https://github.com/candrepa1/take-home-assignment.git. 
* Switch to the final branch with git checkout final.
* Run npm install to add the dependencies and after run npm start to see the changes on your local host environment. 
* Try inputting 'This is some text.' as an example. If no mode was selected, it should return the same string. If any mode was selected, it should return it in uppercase or lowercase depending on what you chose.
